### DISCUSSION TITLE
Some nodes failed to load

### DISCUSSION THREAD

**Discussion by robbigoal** [melMass/comfy_mtb#48] [General]
[comfy_mtb] | INFO -> Some nodes failed to load:
        Failed to import module faceswap because More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates

what's the solution ?

**Comment by melMass** [comment]
Please use the issue tracker for that and post the full log + a bit more information about your system and setup:

[Create new issue](https://github.com/melMass/comfy_mtb/issues/new/choose)
